sources/models/src/lib.rs

```diff
@@ -39,6 +39,11 @@ The `#[model]` attribute on Settings and its sub-structs reduces duplication and
 * [Model](src/aws-k8s-1.19/mod.rs)
 * [Default settings](src/aws-k8s-1.19/defaults.d/)
 
+## aws-k8s-1.20: Kubernetes 1.20
+
+* [Model](src/aws-k8s-1.15/mod.rs)
+* [Default settings](src/aws-k8s-1.15/defaults.d/)
```

Comment:
(mentioned in the generated README file, but this is the source - need 1.19 here)